js/react/components/nav/index.jsx

```diff
@@ -47,45 +43,33 @@ function Nav(props) {
 								Community
 							</a>
 						</li>
-						{/* No user */}
-						{!user && (
+						{!user ? (
 							<li className="nav-item">
 								<a className="nav-link" href="/login">
 									Log In
 								</a>
 							</li>
-						)}
-						{/* User gift sender */}
-						{!user?.isAdmin && !user?.userRole === 'partner' && (
+						) : (
 							<li className="nav-item">
 								<a className="nav-link" href="/profile">
 									Profile
 								</a>
 							</li>
 						)}
-						{/* User is admin */}
-						{user?.isAdmin && (
+						{user?.userRole === 'partner' && agency?.isVerified ? (
+							<li className="nav-item">
+								<a className="nav-link" href="/wishcards/create">
+									Create a Wish Card
+								</a>
+							</li>
+						) : null}
+						{user?.userRole === 'admin' ? (
 							<li className="nav-item">
 								<a className="nav-link" href="/admin/">
 									Admin Panel
 								</a>
 							</li>
-						)}
-						{/* User partner and agency is verified */}
-						{user?.userRole === 'partner' && agency?.isVerified && (
-							<>
-								<li className="nav-item">
-									<a className="nav-link" href="/wishcards/me">
-										My Wishes
-									</a>
-								</li>
-								<li className="nav-item">
-									<a className="nav-link" href="/wishcards/create">
-										Create a Wish Card
-									</a>
-								</li>
-							</>
-						)}
+						) : null}
```

Comment:
you can actually remove the null defaults here and change it to a condition chain
```suggestion
						{user?.userRole === 'partner' && agency?.isVerified && (
							<li className="nav-item">
								<a className="nav-link" href="/wishcards/create">
									Create a Wish Card
								</a>
							</li>
						)}
						{user?.userRole === 'admin' && (
							<li className="nav-item">
								<a className="nav-link" href="/admin/">
									Admin Panel
								</a>
							</li>
						)}
```

Reply:
@Enubia  - Oh I try to avoid && chains whenever possible because I was told it's not a good practice by a senior react dev at work.
It's to prevent '0' from showing up. For example, if you do {someData.length && (<></>)} this can show a 0 when the condition does not meet. I was told it's not a good habit, as it can cause unwanted defect in some cases.

Reply:
@stacysealky thanks for sharing that, good to know, https://stackoverflow.com/questions/53048037/react-showing-0-instead-of-nothing-with-short-circuit-conditional-component, will look more into it

Reply:
> @Enubia - Oh I try to avoid && chains whenever possible because I was told it's not a good practice by a senior react dev at work. It's to prevent '0' from showing up. For example, if you do {someData.length && (<></>)} this can show a 0 when the condition does not meet. I was told it's not a good habit, as it can cause unwanted defect in some cases.

Ah ok, wasn't aware of that. Well, it's ugly but I can see the benefit. Please ignore my suggestion in that case.